src/sinks/axiom.rs

```diff
@@ -100,6 +100,7 @@ impl SinkConfig for AxiomConfig {
             query: Some(query),
             tls: self.tls.clone(),
             request,
+            api_version: ElasticsearchApiVersion::V6,
```

Comment:
I know the Axiom integration tests recently broke, it would be good to know what "ES" version we should be aiming to support/track here. (Somewhat unrelated to this change)

Reply:
I've tested axiom with v6, v7, and v8 `api_version`, and all of them passed on the one relevant test `axiom_logs_put_data`. So it would seem axiom isn't taking `_type` into consideration since that's the only difference between the versions. As such it makes sense to keep this version as current as possible, which would be v8 at the moment. But considering that it is only one test, keeping current behavior with v6 is a safer choice by a small margin.